apps/web/src/app/api/checker/regions/_checker.ts

```diff
@@ -121,3 +123,23 @@ export const ping = async (
 
   return res;
 };
+
+const triggerAlerting = async ({ monitorId }: { monitorId: string }) => {
+  const notifications = await db
+    .select()
+    .from(schema.notificationsToMonitors)
+    .leftJoin(
+      schema.notification,
+      eq(schema.notification.id, schema.notificationsToMonitors.notificationId),
+    )
+    .leftJoin(
+      schema.monitor,
+      eq(schema.monitor.id, schema.notificationsToMonitors.monitorId),
+    )
+    .where(eq(schema.monitor.id, Number(monitorId)))
+    .all();
+
+  for (const notif of notifications) {
+    // Send the appropriate notification
```

Comment:
@mxkaske Here i want to call the the send method from the  appropriate package 
e.g  notification.name = 'email' -> call openstatus/resend/send.ts 
notification.name='discord' -> call openstatus/discord/send.ts